product/host.go

```diff
@@ -124,6 +124,17 @@ func hostRunners(ctx context.Context, os string, redactions []*redact.Redact, l
 	}
 	r = append(r, lsMod)
 
+	//Dmesg runner
+	dMesg, err := host.NewDMesg(host.DMesgConfig{
+		OS:         os,
+		Timeout:    TimeoutTenSeconds,
+		Redactions: redactions,
+	})
+	if err != nil {
+		l.Error("unable to create host.Dmesg runner.", "err=", err)
```

Comment:
This is not something we need to fix in this PR, but I just noticed that the code for all the runner-creation here just ignores failures and adds failed runners to the runner list after logging. @nwchandler and @Ayeyiiat should I create a ticket to wrap all of the actual runner-slice appends in an `else` clause? E.g.
```
lsMod, err := host.NewLsmod(host.LsmodConfig{
	OS:         os,
	Timeout:    TimeoutTenSeconds,
	Redactions: redactions,
})
if err != nil {
	l.Error("unable to create host.Lsmod runner.", "err=", err)
} else {
	r = append(r, lsMod)
}
```

Reply:
Good catch, @groovemonkey! Yeah, if you could create a ticket for that, I think it would be worthwhile. Failures in any of these, right now, look like they'd probably cause a nil pointer panic.